packages/financial-templates-lib/src/price-feed/PriceFeedMock.js

```diff
@@ -1,32 +1,46 @@
+const { toBN } = web3.utils;
 const { PriceFeedInterface } = require("./PriceFeedInterface");
+const { parseFixed } = require("@ethersproject/bignumber");
 
 // An implementation of PriceFeedInterface that medianizes other price feeds.
 class PriceFeedMock extends PriceFeedInterface {
   // Constructs the MedianizerPriceFeed.
   // priceFeeds a list of priceFeeds to medianize. All elements must be of type PriceFeedInterface. Must be an array of
   // at least one element.
-  constructor(currentPrice, historicalPrice, lastUpdateTime) {
+  constructor(currentPrice, historicalPrice, lastUpdateTime, invertPrice, decimals = 18) {
     super();
     this.updateCalled = 0;
     this.currentPrice = currentPrice;
     this.historicalPrice = historicalPrice;
     this.lastUpdateTime = lastUpdateTime;
     this.historicalPrices = [];
+    this.invertPrice = invertPrice;
+
+    this.convertDecimals = number => {
+      // Converts price result to wei
+      // returns price conversion to correct decimals as a big number
+      return toBN(parseFixed(number.toString(), decimals).toString());
```

Comment:
I know we've discussed this elsewhere but what do you guys think about _not_ using the `ethers`  `parseFixed` function and rather implementing it ourselves using web3.js so that we don't need to use both ethers _and_ web3.js in the bots? I'm really not a fan of needing to import _both_ (I know web3 uses ethers under the hood but still).

Reply:
i did originally have my own version of this, but ended up going with ethers implementation because i was a bit cleaner and also handles decimals.